frontend/src/metabase/components/List/List.css

```diff
@@ -133,28 +138,29 @@
   visibility: visible !important;
   margin-left: 10px;
 }
+
 :local(.item):hover :local(.itemCheckbox),
 :local(.item.selected) :local(.itemCheckbox) {
   display: inline;
 }
+
 :local(.item.selected) :local(.itemCheckbox) {
   color: var(--color-brand);
 }
 
 /* ITEM ICON */
 :local(.itemIcon) {
-  composes: icon;
   visibility: visible !important;
   composes: relative from "style";
 }
+
 :local(.item):hover :local(.itemIcon),
 :local(.item.selected) :local(.itemIcon) {
   display: none;
 }
 
 /* CHART ICON */
 :local(.chartIcon) {
-  composes: icon;
```

Comment:
You can't remove `composes`. Seems like duplicate `composes` works in CSS module.

You can see this by visiting `/reference/databases`

#### This PR
![image](https://github.com/metabase/metabase/assets/1937582/64c0c188-955a-4c69-8137-63ac6288805e)


#### bringing `composes: icon;` back
![image](https://github.com/metabase/metabase/assets/1937582/abdb6cd4-3fa8-447e-a420-fbcf98317264)

I found that if you place `composes` next to each other, stylelint won't complain about that e.g.
https://github.com/metabase/metabase/blob/52edd3d124f8ab52d1bc5d2e77043da3482169bf/frontend/src/metabase/components/List/List.css#L172-L173